config/site_settings.yml

```diff
@@ -575,6 +575,9 @@ groups:
   enable_category_group_review:
     client: true
     default: false
+  group_list_changed:
```

Comment:
I support something here, but using a site setting for feature detection is a bit overboard imo.

Technically what happened here is that this API went up: 

https://github.com/discourse/discourse/blob/b9a2a4d904ee28ff458fe34577e433ca719b2174/app/assets/javascripts/discourse/lib/plugin-api.js.es6#L47-L48

So we are now at 0.8.32

So bump that up, and figure out a way of letting the server know about this as well, something reasonably crude can work here like splitting off PLUGIN_API_VERSION into a dedicated JS file and then having `Discourse.client_plugin_api` read the file.

This makes it far cleaner to make decisions in plugins.

Reply:
The server will always return IDs. The plugin API version will be used by plugins to assume which value they are dealing with (IDs or names), and also decide if they have to migrate.

All new code using this type of site setting must assume IDs are going to be stored.